packages/react/src/components/Forms/DatePicker/index.mdx

```diff
@@ -27,6 +28,12 @@ You can use the following props to change the default value of the DatePicker co
   <Story id="components-forms-datepicker--default-value"/>
 </Canvas>
 
+## Range
+
+<Canvas>
+  <Story id="components-forms-datepicker--range-default"/>
+</Canvas>
+
 
```

Comment:
IMO we should mention how the formData are fulfilled by explaining the _start _end convention

Reply:
agree